packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -0,0 +1,169 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+@mixin spinner-size($inner-size, $outer-size) {
+  width: $outer-size;
+  height: $outer-size;
+
+  &::before,
+  &::after {
+    border-width: ($inner-size / 8);
+    width: $inner-size;
+    height: $inner-size;
+    $padding: (($outer-size - $inner-size) / 2);
+    top: $padding;
+    left: $padding;
+  }
+}
+
+/*
+Spinner
+
+Spinners signify that the application is waiting for an asychronous operation to complete.
+
+.ds-c-spinner
+
+Markup:
+<span class="ds-c-spinner ds-c-spinner--small"></span>
+<span class="ds-c-spinner"></span>
+<span class="ds-c-spinner ds-c-spinner--big"></span>
+<br>
+<span class="ds-c-spinner ds-u-color--primary"></span>
+<span class="ds-c-spinner ds-u-color--success"></span>
+<span class="ds-c-spinner ds-u-color--muted"></span>
```

Comment:
These aren't actual variants of the component. Notice it's `ds-u-color-*`. I'm just giving examples of using color utility classes to modify the colors. I can remove them though.